### PR TITLE
Shared Blocklist (minus Appeal)

### PR DESCRIPTION
**Pull request by infinite-persistence**
## 🟡 Status
2021-10-08:  In draft mode until Commentron issues are fixed

## Details
https://www.notion.so/lbry/Shared-Blocklist-Review-3337701e8c68494d92e9cde6b73b2c83
<sub><em>Sorry for the redirect, but it's way easier to type in notion</em></sub>

Closes [#6570 Shared Blocked Lists ](https://github.com/lbryio/lbry-desktop/issues/6570)

## Test
https://kp.odysee.com

### CODE REVIEW

**Comment by infinite-persistence** [comment]
### 463abc0 to f8754fb
- Added help section. TODO: need link to "learn more".
- Fixed flickering.
- Removed emojis in text area for Edit and Invite

**Comment by infinite-persistence** [comment]
#### f8754fb to d6935e3
- Disable "Invite" button for own SBL when you have accepted another SBL.
- Make invitation section collapsed by default, showing only From and Message at first.
- Show status on members (accepted or not)
- Rebased

File: ui/constants/icons.js [hunk 1]
```diff
@@ -171,6 +171,7 @@ export const STAR = 'star';
 export const MUSIC = 'MusicCategory';
 export const BADGE_MOD = 'BadgeMod';
 export const BADGE_STREAMER = 'BadgeStreamer';
+export const APPEAL = 'JudgeHammer';
```
Comment (jessopb) [inline ui/constants/icons.js:174]: I like this USAGE = 'description' pattern.

File: ui/page/appealListOffences/view.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import * as MODALS from 'constants/modal_types';
+
+type Props = {
+  doOpenModal: (id: string, {}) => void,
+};
+
+export default function AppealListOffences(props: Props) {
+  const { doOpenModal } = props;
+
+  const blockerUris = [
```
Comment (jessopb) [inline ui/page/appealListOffences/view.jsx:16]: What are these and why are they hardcoded.. needs comment?

Comment (jessopb) [reply ui/page/appealListOffences/view.jsx:16]: Oh, ,"minus appeal".

File: ui/scss/component/_block-list.scss [hunk 3]
```diff
@@ -83,3 +83,55 @@
...
+}
+
+.sbl_invite_expired {
+  align-self: flex-end;
+  color: var(--color-error);
+}
+
+.sbl_invite_accepted {
+  align-self: flex-end;
+  color: #2bbb90;
```
Comment (jessopb) [inline ui/scss/component/_block-list.scss:122]: no var for this color?

**Review by jessopb — APPROVED** [review]
just a couple non-blocking questions.

File: ui/scss/component/_block-list.scss (hunk 3 above)
Comment (infinite-persistence) [reply ui/scss/component/_block-list.scss:122]: Oops, slipped.

**Comment by infinite-persistence** [comment]
`--- Close ---`
Moving to Odysee repo

**Pull request closed without merging**